ecommerce/src/main/java/app/controller/HomeController.java

```diff
@@ -5,9 +5,14 @@
 
 @Controller
 public class HomeController {
-	
+
 	@GetMapping("/")
 	public String index() {
 		return "views/user/home/index";
 	}
+
+	@GetMapping("/login")
+	public String login() {
+		return "views/user/login/index";
```

Comment:
```suggestion
		return "views/session/login";
```
login logout nó nằm ở folder session chứ ko phải user bé nha

Reply:
ngoài ra, viết nó ở sessionsController đi em